docs/docs/guides/integrate/login-ui/saml-standard.mdx

```diff
@@ -0,0 +1,140 @@
+
+:::note
+The user id sent in the 'x-zitadel-login-client' has to match to the PAT you are sending in the request.
+:::
+
+Read more about the [SSO Endpoint Documentation](/docs/apis/saml/endpoints#sso_endpoint)
+
+The endpoint will redirect you to the domain of your UI on the path /login and add the SAML Request ID as parameter.
+```https://login.example.com/login?authRequest=V2_224908753244265546```
+
+### Get SAML Request by ID
+
+With the ID from the redirect before you will now be able to get the information of the SAML request.
+[Get SAML Request By ID Documentation](/docs/apis/resources/saml_service_v2/saml-service-get-saml-request)
+
+```bash
+curl --request GET \
+  --url https://$ZITADEL_DOMAIN/v2/saml/saml_requests/V2_224908753244265546 \
+  --header 'Authorization: Bearer '"$TOKEN"''\
+```
+
+Response Example:
+
+```json
+{
+	"samlRequest": {
+		"id": "V2_224908753244265546",
+		"creationDate": "2023-07-28T13:47:43.471505Z",
+		"issuer": "https://myapp.example.com",
+    "assertionConsumerService": "https://myapp.example.com/acs",
+    "relayState": "CncN92gdF6is7bak63thXOsn0MmJn7CLQeGKWaXZo2L8nJN0sPEHbb4I",
+    "binding": "urn:oasis:names:tc:SAML:2.0:bindings:HTTP-Redirect"
+	}
+}
+```
+
+### Perform Login
+
+After you have initialized the SAML flow you can implement the login.
+Implement all the steps you like the user to go trough by [creating](/docs/apis/resources/session_service_v2/session-service-create-session) and [updating](/docs/apis/resources/session_service/session-service-set-session) the user-session.
+
+Read the following resources for more information about the different checks:
+- [Username and Password](./username-password)
+- [External Identity Provider](./external-login)
+- [Passkeys](./passkey)
+- [Multi-Factor](./mfa)
+
+### Finalize SAML Request
+
+To finalize the SAML request and connect an existing user session with it you have to update the SAML request with the session token.
+On the create and update user session request you will always get a session token in the response.
+
+The latest session token has to be sent to the following request:
+
+Read more about the [Finalize SAML Request Documentation](/docs/apis/resources/saml_service_v2/saml-service-create-response)
+
+Make sure that the authorization header is from the same account that you originally sent in the client id header ```x-zitadel-login-client: <userid>``` on the SSO endpoint.
+```bash
+curl --request POST \
+  --url $ZITADEL_DOMAIN/v2/saml/saml_requests/V2_224908753244265546 \
+  --header 'Accept: application/json' \
+  --header 'Authorization: Bearer '"$TOKEN"''\
+  --header 'Content-Type: application/json' \
+  --data '{
+  "session": {
+    "sessionId": "225307381909694508",
+    "sessionToken": "7N5kQCvC4jIf2OuBjwfyWSX2FUKbQqg4iG3uWT-TBngMhlS9miGUwpyUaN0HJ8OcbSzk4QHZy_Bvvv"
+  }
+}'
+```
+
+In the response you will get content which describe how to handle further steps from your login UI.
```

Comment:
I'd describe the step(s) a bit in detail and note both cases (redirect and post)

Reply:
I added 2 examples for the 2 types of binding.

Reply:
i fixed the formatting and improved the wording a bit

Reply:
Thanks